xclim/data/fr.json

```diff
@@ -1176,11 +1176,6 @@
     "title": "Température radiative moyenne",
     "abstract": "La température moyenne des rayonnements solaires et thermiques incidents sur un corps à l'extérieur."
   },
-  "RAIN_SEASON": {
-    "description": "Saison des pluies",
-    "title": "Saison des pluies",
-    "abstract": "La saison des pluies commence lorsque deux conditions sont réunies : 1) Il doit y avoir un certain nombre de jours humides avec précipitations supérieures ou égales à un seuil donné ; 2) Il doit y avoir une autre séquence suivante, où, pendant une période de temps donnée, il n'y a pas de séquence sèche (c'est-à-dire un certain nombre de jours où les précipitations sont inférieures ou égales à un certain seuil). La saison des pluies se termine lorsqu'il y a une séquence sèche"
-  },
```

Comment:
Is this not needed as a base for the other `RAIN_SEASON.*` indicators?

Reply:
Yes, I was trying some things to make it work. I don't know how to test this locally?

Reply:
I've already added it back, but I still have the issue with ["", "", "day"]

Reply:
Hmm I still get incomplete translation:
```
FAILED xclim/testing/tests/test_locales.py::test_xclim_translations[fr] - Failed: 0 indicators are missing translations and RAIN_SEASON have incomplete translations for official locale `fr`.
```